diffrax/_brownian/tree.py

```diff
@@ -378,6 +412,21 @@ def _brownian_arch(
          there for the sake of a future extension with "space-time-time" Levy area
          and should be None for now.
 
+         ??? cite "Reference"
+        Based on section 6.1 of
+        ```bibtex
+        @phdthesis{foster2020a,
+          publisher = {University of Oxford},
+          school = {University of Oxford},
+          title = {Numerical approximations for stochastic differential equations},
+          author = {Foster, James M.},
+          year = {2020}
+        }
+
+        In particular see Theorem 6.1.6.
+        ```
+
+
```

Comment:
This should be formatted as:
```
??? cite "References"

    ```bibtex
    ...
    ```
```
i.e. with a new line, with an indent, and with only a single blank line between the reference section and the arguments section.

Reply:
Oops, not sure what happened with the indent there. Will fix